web/html/src/package.json

```diff
@@ -1,33 +1,8 @@
 {
-  "name": "susemanager-nodejs-sdk-devel",
+  "name": "spacewalk-web",
   "version": "1.0.0",
   "description": "Node.js software needed by SUSE Manager at build time.",
```

Comment:
I guess this description is not correct.

Reply:
Definitely not right! eheh